internal/vulnstore/postgres/getupdateoperations.go

```diff
@@ -0,0 +1,150 @@
+package postgres
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/rs/zerolog"
+
+	"github.com/quay/claircore/libvuln/driver"
+)
+
+// GetLatestUpdateRef implements driver.Updater.
+func (s *Store) GetLatestUpdateRef(ctx context.Context) (uuid.UUID, error) {
```

Comment:
Is this used? Libvuln calls store.GetLatestUpateRef, and that calls the private method below this

Reply:
This will be used by clair for conditional requests. For example, a client could issue a GET with `If-None-Match: <validator>`, and if we use the absolute latest update operation as that validator, we can know there's no possible way the result has changed, and short-circuit the response.